content/docs/atom-guide/features/_index.md

```diff
@@ -96,20 +96,20 @@ Atom has a unified mesh format with support for several mesh types:
 *Related to: [Mesh component](/docs/user-guide/components/reference/atom/mesh), [Mesh Feature Processor API](/docs/api/gems/Atom/class_a_z_1_1_render_1_1_mesh_feature_processor.html), [Skinned Mesh Feature Processer API](/docs/api/gems/Atom/class_a_z_1_1_render_1_1_skinned_mesh_feature_processor_interface.html)*  
 
 
-## Processing and Post FX
+## Post-processing effects
 
 Tone Mapping and Color Grading   
 : Techniques that adjust the overall lighting and color in a scene. Tone mapping simulates high dynamic range (HDR) by mapping one set of colors to another set that has a higher range. Color grading adjusts the color output of the render. There are two color grading modes: HDR Color Grading and LDR Color Grading. Tone mapping and color grading can be configured in O3DE using the **Display Mapper**. You can also view a demo of tone mapping and color grading in the Atom Sample Viewer. 
 
 *Related to: [Display Mapper component](/docs/user-guide/components/reference/atom/display-mapper/), [Tonemapping (Atom Sample Viewer)](/docs/atom-guide/atom-sample-viewer/graphics-feature-samples.md#tonemapping)*
 
 Post-processing Volumes   
-: Volumes that allow PostFX to be bounded in certain shapes and areas in the game. Post-processing volumes are integrated into O3DE via a shape component and one of the PostFX components.
+: PostFX can be weighted within a volume or area by a shape, gradient, or radius. In O3DE, this is done with a shape that is referenced by a **PostFX Shape Weight Modifier** component, with a gradient that is referenced through a **PostFX Gradient Weight Modifier** component, or by distance through a **PostFX Radius Weight Modifier** component.
```

Comment:
we should always use 'gradient signal' and/or 'gradient signal entity' instead of just 'gradient'?

Reply:
Honestly, I think you'd have better understanding by making a bullet list (maybe something like):

The blending of PostFX layers can be controlled using various mechanisms in O3DE, each layer can be weighted within a volume or area by a shape, gradient, or radius.
- A simple distance based blend can be set up by adding the **PostFX Radius Weight Modifier** component
- A volume+falloff based blend is set up by adding a Shape Entity plus the **PostFX Shape Weight Modifier**
- A more complex blend can be configured by using a gradient signal that is referenced through a **PostFX Gradient Weight Modifier** as a masking operation.

Me speaking:  This outlines simple to complex and helps me understand the purpose and use case of each more easily.